dashboard/app/views/pd/application/teacher_application_mailer/declined.html.haml

```diff
@@ -9,12 +9,15 @@
   = @application.course_name + "."
   Our team appreciated the opportunity to review your application and learn about your
   commitment to computer science education. I'm writing to let you know that we are currently
-  unable to offer you a space in the Professional Learning Program this year.
+  unable to offer you a space in the Professional Learning Program this year. We encourage you
+  to explore our
+  = link_to('free, self-paced online Professional Learning', 'https://code.org/educate/professional-development-online')
+  programs.
 
 %p
   We hope that you will continue to use Code.org's resources and materials
-  which are available free for all educators at
-  = link_to('code.org/educate', 'https://code.org/educate') + '.'
+  which are free for all educators at
+  = link_to('code.org/teach', 'https://code.org/teach') + '.'
```

Comment:
Not blocking: does there need to be two links here, or can you just use `https://code.org/teach`?

Reply:
In the preview, it doesn't appear hyperlinked. It wouldn't surprise me if, in an actual email client, it is hyperlinked, but I think I'll stick with the explicit link just in case.

<img width="1395" alt="Screenshot 2024-01-18 at 8 06 11 AM" src="https://github.com/code-dot-org/code-dot-org/assets/46464143/4bbe3184-a6ef-4737-aa58-3c0a4f3f3f0a">